counsel.el

```diff
@@ -5267,21 +5283,35 @@ trees."
                                 (match-string-no-properties 1)))))))
     (sort targets #'string-lessp)))
 
+(defun counsel-compile--pretty-propertize (leader text face)
+  "Return a pretty string of the form ' LEADER TEXT '
+  where LEADER is propertized with a warning face and the remaining
+  text with FACE."
```

Comment:
Please indent and format this properly and separate the summary from the rest of the text, e.g.:
```el
  "Return a pretty string of the form \" LEADER TEXT \".
LEADER is propertized with a warning face and the remaining text
with FACE."
```